apps/users/models.py

```diff
@@ -77,12 +84,24 @@ class Student(Base):
     middle_name = Column(VARCHAR(length=50))
     telephone_number = Column(VARCHAR(length=50), nullable=False, unique=True)
     gender = Column(VARCHAR(length=1), nullable=False)
-    course_id = Column(INTEGER, ForeignKey("course.course_id"), nullable=False)
+    course_id = Column(
+        INTEGER, ForeignKey("course.course_id", ondelete="CASCADE", onupdate="CASCADE"),
+        nullable=False,
+    )
     speciality_id = Column(
-        INTEGER, ForeignKey("speciality.speciality_id"), nullable=False
+        INTEGER,
+        ForeignKey("speciality.speciality_id", ondelete="CASCADE", onupdate="CASCADE"),
+        nullable=False,
+    )
+    user_id = Column(
+        INTEGER,
+        ForeignKey("user.user_id", ondelete="CASCADE", onupdate="CASCADE"),
+    )
+    faculty_id = Column(
+        INTEGER,
+        ForeignKey("faculty.faculty_id", ondelete="CASCADE", onupdate="CASCADE"),
```

Comment:
ondelete value isn't correct, i think it should be do nothing or set null...

Reply:
The same in previous question. ?